src/client/client4.js

```diff
@@ -2603,5 +2603,7 @@ export class ClientError extends Error {
         this.intl = data.intl;
         this.server_error_id = data.server_error_id;
         this.status_code = data.status_code;
+
```

Comment:
Is it worth adding a comment here explaining the context? Perhaps something like:
```suggestion

// Ensure message is treated as a property of this class when object spreading.
```

Reply:
Good call. I'll add that